dev/vagrant/bootstrap/vault/post/01-create-vault-policy-to-read-secrets.yml

```diff
@@ -7,8 +7,7 @@
     EOF
   register: cmd_op
   environment:
-      VAULT_ADDR: "{{ lookup('env', 'VAULT_ADDR') }}"
       VAULT_TOKEN: "{{ lookup('env', 'vault_master_token') }}"
```

Comment:
Is not `VAULT_TOKEN` available just like `VAULT_ADDR`?

Reply:
Logs says NO.

```
TASK [Debug all env vars] ******************************************************
ok: [default] => {
    "ansible_env": {
        "ANSIBLE_FORCE_COLOR": "true",
        "HOME": "/root",
        "LANG": "en_US.UTF-8",
        "LANGUAGE": "en_US:",
        "LOGNAME": "root",
        "MAIL": "/var/mail/vagrant",
        "OLDPWD": "/home/vagrant",
        "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin",
        "PWD": "/etc/ansible",
        "PYTHONUNBUFFERED": "1",
        "SHELL": "/bin/bash",
        "SHLVL": "2",
        "SSH_CLIENT": "10.0.2.2 52425 22",
        "SSH_CONNECTION": "10.0.2.2 52425 10.0.2.15 22",
        "SUDO_COMMAND": "/bin/sh -c echo BECOME-SUCCESS-cybochfzwuzzoxdidpahgydidbjyysvi ; /usr/bin/python3 /home/vagrant/.ansible/tmp/ansible-tmp-1605510961.770645-2166-3289445587776/AnsiballZ_setup.py",
        "SUDO_GID": "1000",
        "SUDO_UID": "1000",
        "SUDO_USER": "vagrant",
        "TERM": "vt100",
        "TF_VAR_consul_acl": "true",
        "TF_VAR_consul_acl_default_policy": "allow",
        "TF_VAR_consul_enterprise": "false",
        "TF_VAR_consul_master_token": "master",
        "TF_VAR_nomad_acl": "false",
        "TF_VAR_nomad_acl_master_token": "master",
        "TF_VAR_nomad_enterprise": "false",
        "TF_VAR_vault_enterprise": "false",
        "TF_VAR_vault_master_token": "master",
        "TF_VAR_vault_pki": "true",
        "USER": "root",
        "USERNAME": "root",
        "VAULT_ADDR": "http://127.0.0.1:8200",
        "XDG_DATA_DIRS": "/usr/local/share:/usr/share:/var/lib/snapd/desktop",
        "XDG_RUNTIME_DIR": "/run/user/1000",
        "XDG_SESSION_ID": "3",
        "_": "/usr/local/bin/ansible-playbook",
        "consul_acl": "true",
        "consul_acl_default_policy": "allow",
        "consul_enterprise": "false",
        "consul_master_token": "master",
        "nomad_acl": "false",
        "nomad_acl_master_token": "master",
        "nomad_enterprise": "false",
        "vault_enterprise": "false",
        "vault_master_token": "master",
        "vault_pki": "true"
    }
}
```

Reply:
Oh right, we removed that a little while back, didn't we. Sorry!

Reply:
Do not apologize. It is good to remind.